pkg/provider/kubernetes/featuregates_test.go

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
this is the wrong place for testing API. It should be here: `kubermatic/pkg/handler/v2/feature_gates/feature_gates_test.go`